monitor_other_boards.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+from datetime import datetime
+import subprocess
+
+import slack_reporter
+
+#                SSH target               name
+other_boards = (("odroid@odroid",         "odroid-C4"),
+                ("orangepi@oliviaorange", "orange pi zero 2"),
```

Comment:
I should double-check my capitalization and spacing in these names. It'll bug me if it's spelled differently in different Slack messages.